cpp/cmake/rapids_config.cmake

```diff
@@ -26,10 +26,12 @@ else()
   )
 endif()
 
+set(rapids-cmake-repo KyleFromNVIDIA/rapids-cmake)
+set(rapids-cmake-branch nvcomp-nested-dir)
 if(NOT EXISTS "${CMAKE_CURRENT_BINARY_DIR}/KVIKIO_RAPIDS-${RAPIDS_VERSION_MAJOR_MINOR}.cmake")
   file(
     DOWNLOAD
-    "https://raw.githubusercontent.com/rapidsai/rapids-cmake/branch-${RAPIDS_VERSION_MAJOR_MINOR}/RAPIDS.cmake"
+    "https://raw.githubusercontent.com/KyleFromNVIDIA/rapids-cmake/nvcomp-nested-dir/RAPIDS.cmake"
```

Comment:
This is to get the changes in rapidsai/rapids-cmake#675 and will be removed once that is merged.